.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - run: echo "no operation build"
+
+workflows:
+  build:
+    jobs:
+      - build
```

Comment:
This file seems like a dummy file of some kind commited by mistake. If you need to add CI logic, go for github action based logic.

Reply:
I had added it because circleci was throwing an error related to not finding a config file when I submitted the PR. I think I copied it from the main branch.  So it should just be removed altogether?

Reply:
Yeah remove it, ill absolutely ignore such error and not let that block merge.